.github/workflows/docker.yaml

```diff
@@ -49,43 +49,57 @@ jobs:
       with:
         authToken: '${{ secrets.CACHIX_CARDANO_SCALING_AUTH_TOKEN }}'
 
-    - name: 🔨 Build image using nix
+    - name: 🔨 🏷️ Build and tag images
       run: |
+
         IMAGE_NAME=ghcr.io/${{github.repository_owner}}/${{matrix.target}}
         echo "IMAGE_NAME=${IMAGE_NAME}" >> $GITHUB_ENV
         nix build .#docker-${{ matrix.target }}
         docker load < result
 
-        # Determine whether we are building a tag and if yes, set a VERSION_NAME
+        # Compute tags. We want the following behaviour:
+        #
+        # 1. Building on tag:      image tagged `tag`
+        # 2. Building on release:  image tagged `release`
+        # 3. Building on master:   image tagged `unstable`.
```

Comment:
I think this should be mentioned in our [CONTRIBUTING](https://github.com/cardano-scaling/hydra/blob/master/CONTRIBUTING.md#versioning-and-changelog) guide.